boards/feather_m4/examples/adc.rs

```diff
@@ -0,0 +1,65 @@
+#![no_std]
+#![no_main]
+
+use feather_m4 as bsp;
+
+use bsp::hal;
+use bsp::pac;
+
+#[cfg(not(feature = "use_semihosting"))]
+use panic_halt as _;
+#[cfg(feature = "use_semihosting")]
+use panic_semihosting as _;
+
+use bsp::entry;
+use bsp::Pins;
+use pac::{CorePeripherals, Peripherals};
+
+use hal::{
+    adc::{Accumulation, Adc, Config, Prescaler, Resolution},
+    clock::v2::{clock_system_at_reset, pclk::Pclk},
+};
+
+#[entry]
+fn main() -> ! {
+    let mut peripherals = Peripherals::take().unwrap();
+    let _core = CorePeripherals::take().unwrap();
+
+    let pins = Pins::new(peripherals.port);
+
+    // TODO: currently this example only runs at the chip's
```

Comment:
Let's reference https://github.com/atsamd-rs/atsamd/issues/819 in the comment (presuming this is the referenced bug!)

Reply:
I think we can now remove the comment and either leave the clocks in their reset state, or feed a clock up to 100 MHz to the ADC.

Reply:
Same with the `metro_m4` ADC examples.

Reply:
Fixed in https://github.com/rnd-ash/atsamd/pull/10